### ISSUE TITLE
Safari: Images are disproportionate

### ISSUE THREAD

**Issue by KateChuen** [SAP/spartacus#7817]
**Describe the bug**
Images are disproportionate

**To Reproduce**
1. Go to homepage
<img width="1412" alt="Screen Shot 2020-05-29 at 9 45 59 AM" src="https://user-images.githubusercontent.com/45409515/83268447-a47ad800-a193-11ea-86b1-435b3fa2593b.png">

2. Go to added-to-cart modal
<img width="833" alt="Screen Shot 2020-05-29 at 9 51 58 AM" src="https://user-images.githubusercontent.com/45409515/83268474-b0ff3080-a193-11ea-81f9-64b58ede00cc.png">

3. Go to cart page
<img width="1370" alt="Screen Shot 2020-05-29 at 9 57 21 AM" src="https://user-images.githubusercontent.com/45409515/83268511-bceaf280-a193-11ea-90fe-1d9c0a2698e6.png">

**Desktop (please complete the following information):**
 - OS: Catalina 10.15.4
 - Browser: Safari 13.1

**Additional context**
release/2.0.0-rc.4.
It happens against both 1905 and 2005.

**Comment by tobi-or-not-tobi** [comment]
@KateChuen is this safari only?

**Comment by KateChuen** [comment]
Yep, safari only.

**Comment by giancorderoortiz** [comment]
I don't see a fix yet. 
Deployed localhost latest develop branch (June 1st)
Was it merged to develop or some other release branch?

**Comment by giancorderoortiz** [comment]
Attaching some images showing the problem described here
![Screen Shot 2020-06-01 at 6 30 31 AM](https://user-images.githubusercontent.com/46171897/83400902-b0ec7400-a3d1-11ea-8adc-3a9a51a31285.png)
![Screen Shot 2020-06-01 at 6 30 41 AM](https://user-images.githubusercontent.com/46171897/83400904-b1850a80-a3d1-11ea-8c8b-86f2bbc43aa4.png)
![Screen Shot 2020-06-01 at 6 32 04 AM](https://user-images.githubusercontent.com/46171897/83400908-b1850a80-a3d1-11ea-94b8-2acc29bdee77.png)

**Comment by giancorderoortiz** [comment]
![Screen Shot 2020-06-01 at 6 34 05 AM](https://user-images.githubusercontent.com/46171897/83401000-eb561100-a3d1-11ea-9037-2366347e3cf9.png)

**Comment by giancorderoortiz** [comment]
Redeployed localhost and dev-server. 
I see the fix. Setting to done.

**Comment by giancorderoortiz** [comment]
![Screen Shot 2020-06-01 at 9 24 45 AM](https://user-images.githubusercontent.com/46171897/83413446-d71e0e00-a3e9-11ea-9c54-0a1f15dfc7b3.png)
![Screen Shot 2020-06-01 at 9 24 59 AM](https://user-images.githubusercontent.com/46171897/83413448-d7b6a480-a3e9-11ea-8c7f-ce84c45aa6e3.png)

**Comment by blaiseup** [comment]
The fix is missing from release 2.0.4

In `node_modules/@spartacus/styles/scss/components/content/media/_media.scss`:
```
%cx-media {
  position: relative;
  display: flex;
  align-items: stretch;
```
